README.md

```diff
@@ -53,7 +57,7 @@ weather === "sunny";
 
 What are the values of these expressions?  Explain your answers.
 
-<!-- Replace this comment with your answer -->
+In the first line, the variable weather is undefined. In the second line, there is one equal sign used, which means it has been assigned the value of "sunny". In the third line, it should evaluate to "true" because the use of three equal signs means that is a strict equality (meaning is it trying to see if the value of weather is equal to "sunny").
```

Comment:
Yes, this is right. It should be noted, though, that `weather = "sunny";` is also an expression, and it also evaluates to a value (in this case, "sunny").